ethereum/p2p/src/main/java/org/hyperledger/besu/ethereum/p2p/discovery/internal/PeerDiscoveryController.java

```diff
@@ -436,10 +436,10 @@ private Optional<PeerInteractionState> matchInteraction(final Packet packet) {
   private void refreshTableIfRequired() {
     final long now = System.currentTimeMillis();
     if (lastRefreshTime + tableRefreshIntervalMs <= now) {
-      LOG.debug("Peer table refresh triggered by timer expiry");
+      LOG.debug("Refreshing peer table after {}ms", tableRefreshIntervalMs);
       refreshTable();
     } else if (!peerRequirement.hasSufficientPeers()) {
-      LOG.debug("Peer table refresh triggered by insufficient peers");
+      LOG.debug("Refreshing peer table: seeking more peers");
```

Comment:
I like that the reasons why were stated. 
What about "Peer requirement of {} not met. Refreshing peer table: seeking more peers"?

Reply:
do you mean logging maxPeers? it's not that easy - we don't have the maxPeers number here

Reply:
I think it would be good to express why the hasSufficientPeers condition wasn't met, whatever that is.

Reply:
How about:
Refreshing peer table: seeking more peers. peer count < max

Reply:
I think "peer requirement" only makes sense if you read the code

Reply:
Agree, I misunderstood: I was interpreting peerRequirement as min peers, but if the condition is peer count < max then  what you said makes sense. Can we print the current peer count here (if not the maxPeers)?

Reply:
not easily :/

Reply:
nw, what you have is definitely an improvement :)